haiku-apps/sum_it/sum_it-0.2beta.recipe

```diff
@@ -5,10 +5,8 @@ Sum-It, the open source version of Maarten Hekkelman's Sum-It application. \
 Sum-It has many features, including a plug-in architecture for formulas and \
 graphs. This product includes software developed by Hekkelman Programmatuur B.V."
 HOMEPAGE="https://github.com/HaikuArchives/OpenSumIt"
-COPYRIGHT="
-	1996-1998, 2000	Hekkelman Programmatuur B.V.  All rights reserved.
-	1999-2000 Tim Vernum. All rights reserved.
-	"
+COPYRIGHT="1996- 2000Hekkelman Programmatuur B.V.  All rights reserved.
```

Comment:
Hmmm...I think this fix is going to have to be manual because there was a tab originally